model/src/main/kotlin/ScannerRun.kt

```diff
@@ -27,8 +27,7 @@ import java.time.Instant
 import org.ossreviewtoolkit.model.config.ScannerConfiguration
 import org.ossreviewtoolkit.model.utils.ProvenanceResolutionResultSortedSetConverter
 import org.ossreviewtoolkit.model.utils.ScanResultSortedSetConverter
-import org.ossreviewtoolkit.model.utils.alignRevisions
-import org.ossreviewtoolkit.model.utils.clearVcsPath
+import org.ossreviewtoolkit.model.utils.getKnownProvenancesWithoutVcsPath
```

Comment:
Nit: This commit could nicely be squashed with the previous one, saying to move two extension functions from `ScannerRun` to `Extensions` (with the same rationale, that then only needs to be mentioned once in the single commit message).